### ISSUE TITLE
problems with admin theme on Medium devices

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/e107#2578]
today update, cache cleared, I used bootstrap breakpoints
 
Medium devices:
![image](https://cloud.githubusercontent.com/assets/5429548/25070116/1af8231a-2295-11e7-93e5-e18930cbc12a.png)

Large devices looks correct:
![image](https://cloud.githubusercontent.com/assets/5429548/25070123/6ea3d3ba-2295-11e7-94c7-e7045248141e.png)

Small devices too:
![image](https://cloud.githubusercontent.com/assets/5429548/25070126/87ce0eb4-2295-11e7-9709-a8632d5eb28b.png)

**Comment by CaMer0n** [comment]
1100px width is the breakpoint, and it is necessary due to everything that is included in the navbar.

**Comment by Jimmi08** [comment]
that wasn't point. 
Small devices has text with icons, but medium devices not?  Dropdown only with icons?

**Comment by CaMer0n** [comment]
It's using "hidden-md hidden-lg" in the admin_template.php file on line 30 and 42. I'm not sure why it is displaying differently to the nav-collapse . Do you?

**Comment by Jimmi08** [comment]
Yes, I know, in fact now I always need to  delete these classes to have texts on md and lg.  Really I don't understand why to hide them on bigger screens.

**Comment by drasehat** [comment]
maybe add "hidden-sm" ?
bring back the nav title .. bring back the nav title .. LOL

**Comment by Jimmi08** [comment]
Why hidden them? In small they are in the line in the dropdown, enough place. I am getting to used to lg nav, but just by counting - first is prefs, third content... some people just are not able to sense icons with the same colour. New one has to be  confused. 
I am not able to check it now - but if you change breakpoint for mobile dropdown, did you change breakpoint for md too? (Grid). Because changing this only for navbar and then use classes with md it's not good idea.

**Comment by drasehat** [comment]
as i said, i agree with you about re-show the nav title .. it's a beta version though, @CaMer0n will decide based on our feedbacks or make a pref for the nav title (show/hide).
i prefer to wait the 2.1.6 stable version released, before i do something with the main admin..

**Comment by CaMer0n** [comment]
I'll probably go with a pref.

**Comment by rica-carv** [comment]
It would be much more user admin friendly to make it dynamic (ie, show or hide depending of screen size...)

**Comment by Moc** [comment]
Does the issue still occur in the latest version?

**Comment by Moc** [comment]
@Jimmi08 still an issue?
